### DISCUSSION TITLE
Resolving error: Unable to locate file in Vite manifest: resources/css/filament/...

### DISCUSSION THREAD

**Discussion by raaquino** [GuavaCZ/filament-knowledge-base#47] [General]
The documentation did not mention this, but in Filament, you still need to add the line in vite.config.js for example;

import { defineConfig } from 'vite';
import laravel from 'laravel-vite-plugin';

export default defineConfig({
    plugins: [
        laravel({
            input: [
                'resources/css/app.css',
                'resources/css/filament/app/theme.css',
                'resources/js/app.js',
            ],
            refresh: true,
        }),
    ],
});

**Comment by lukas-frey** [comment]
That's not an issue with this package, you have to do that in order to register the custom theme.

Which is a filament feature and in the provided link it's documented [here](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) and the command also notified you after you create the theme. 🙂
